docs/content/index.md

```diff
@@ -54,9 +57,9 @@ file.content "render" {
 }
 ```
 
-Invoke this with `converge apply --local samples/fileContent.hcl` to install
-Traefik from yum on your system. You can also `converge plan --local
-samples/fileContent.hcl` to see what changes will be made before you apply them.
+It is a good idea to `converge plan --local samples/fileContent.hcl` to see what
+changes will be made before you apply them. Invoke this with `converge apply
```

Comment:
How about "If you like what you see, you can install Traefik on your system by running `converge apply …`."?